backend/store.py

```
@@ -675,11 +816,11 @@ def clear_clusters():
 
 
 def set_reddit_subreddits(subreddits: List[str]) -> List[str]:
```

Ruff (0.14.8)

Unused function argument: `subreddits`

(ARG001)